.github/workflows/prerelease.yml

```diff
@@ -1,69 +1,71 @@
-# This workflow will ensure all PRs are labeled so we can later create releases
-
 name: Prerelease
 
 on:
-  - push
-  - workflow_dispatch
+  push:
+    branches-ignore:
+      - "master"
 
 jobs:
   prerelease:
-    if: "github.event_name == 'workflow_dispatch' || contains(github.event.head_commit.message, '[prerelease]')"
+    if: "contains(github.event.head_commit.message, '[prerelease]')"
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: ./.github/actions/setup
+      - name: Run Setup
+        uses: ./.github/actions/setup
         with:
-          npm_token: ${{secrets.npm_token}}
+          npm_token: ${{ secrets.npm_token }}
       - uses: ./.github/actions/git-creds
       - name: Check if there are changed packages
-        id: changed-packages
         run: |
           changed_packages=$(yarn -s lerna ls --since=origin/master --json --loglevel=error)
-          [[ $changed_packages = "[]" ]] && echo "No packages changed in this branch" && exit 1 || echo "Continuing to prerelease"
+          [[ $changed_packages = "[]" ]] && echo "Skipping prerelease. Nothing to prerelease as Lerna didn't detect any changes." && exit 1 || echo "Changed detected. Continuing to prerelease" && echo "$changed_packages"
       - name: Build all affected packages
-        run: yarn lerna run build --since=origin/master
-      - name: Generate prerelease prefix
-        run: |
-          normalized_branch_name=$(git rev-parse --abbrev-ref HEAD | tr -dc '[:alnum:]\n\r' | tr '[:upper:]' '[:lower:]')
-          echo "BRANCH=$normalized_branch_name" >> $GITHUB_ENV
+        run: yarn lerna run build --since=origin/master --include-dependencies
+      - name: Run Tests
+        shell: bash
+        run: yarn lerna run test --since=origin/master
```

Comment:
Why?

Reply:
They used to wait for tests to finish, we can pass on it, but just to be on safe side..

Reply:
I think we can forego tests altogether 🤷‍♂️

Reply:
@talkor mind removing this step plz? 😅🙏🏻

Reply:
Done